README.md

```diff
@@ -23,8 +23,36 @@ refer to the [roadmap] and the issue tracker for more details.
 
 This repository provides pre-built binaries that are signed and published
 using [GoReleaser]. The signature for these binaries are generated using
-[Sigstore], using the release workflow's identity. Please use release v0.1.0 or
-higher, as prior releases were created to test the release workflow.
+[Sigstore], using the release workflow's identity. Make sure you have
+[cosign] installed on your system, then you will be able to securely download
+and verify the gittuf release:
```

Comment:
Can you document here that `ARCH`, `OS`, `VERSION` must be modified as necessary from the snippet below?

Reply:
Makes sense. Let me know if that explanation works for you.